grakn-dist/src/grakn

```diff
@@ -49,6 +49,6 @@ update_classpath_global_var
 
 GRAKN_CONFIG="conf/grakn.properties"
 
-java -cp ${CLASSPATH} -Dgrakn.dir="${GRAKN_HOME}" -Dgrakn.conf="${GRAKN_HOME}/${GRAKN_CONFIG}" -Dgrakn.pidfile=/tmp/grakn.pid ai.grakn.bootup.GraknBootup $@
+java -cp ${CLASSPATH} -Dgrakn.dir="${GRAKN_HOME}" -Dgrakn.conf="${GRAKN_HOME}/${GRAKN_CONFIG}" -Dengine.javaopts="${ENGINE_JAVAOPTS}" -Dgrakn.pidfile=/tmp/grakn.pid ai.grakn.bootup.GraknBootup $@
```

Comment:
where is ENGINE_JAVAOPTS set in the script?

Reply:
It is supplied by the user when invoking grakn via the command line, ie.,:
```
$ ENGINE_JAVAOPTS='-Xdebug ...' ./grakn server start
Starting Storage.....
```